frontend/src/App.tsx

```diff
@@ -36,7 +36,7 @@ export default function App() {
               <PatientsPage userEmail={userEmail} setUserEmail={setUserEmail} />
             }
           >
-            <Route path=":smilePatientId" />
+            <Route path=":patientId" />
```

Comment:
switching away from the smile patient id?

Reply:
Yep. In the Patient Samples view, I figured out a way to have the URL query param == either the `cmoPatientId` or the `dmpPatientId`. For example, see [https://smile-dev.mskcc.org:3006/patients/C-MP76JR](https://smile-dev.mskcc.org:3006/patients/C-MP76JR). Previously this was the `smilePatientId`.